package.json

```diff
@@ -7,15 +7,15 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "prestart": "npm run -s build",
+    "prestart": "dotenvx run -f .dev.vars -- npm run -s build",
     "start": "next start",
     "lint": "biome check .",
     "pages:build": "npx @cloudflare/next-on-pages",
-    "prepreview": "npm run -s pages:build",
-    "preview": "wrangler pages dev --ip 0.0.0.0",
-    "predeploy": "npm run -s pages:build",
+    "prepreview": "dotenvx run -f .dev.vars -- npm run -s pages:build",
+    "preview": "wrangler pages dev --experimental-vectorize-bind-to-prod --ip 0.0.0.0",
+    "predeploy": "dotenvx run -f .dev.vars -- npm run -s pages:build",
```

Comment:
Why put it on all the usages of `pages:build` instead of just putting it on `pages:build` itself? Would deduplicate this at lot.

I think we only need this on the scripts that involve `next` or `vercel`. So that's:

- `dev`
- `build`
- `start`
- `pages:build`

Reply:
Although, the production deployment is relying on `pages:build` at the moment and I'm afraid the build will start to fail there since `.dev.vars` won't exist there.

Reply:
So, either we change the production deploy to call `npx @cloudflare/next-on-pages` directly or keep what you have I guess.

Reply:
Okay, then for now I'll just assume the user will manually prefix the dotenvx command and remove it from all package.json scripts